File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.js
function solution(code) {
    const joinedStr = code.split("1").join('');

    let result = "";
    for (let i = 0; i < joinedStr.length; i += 2) {
        result += joinedStr[i];
    }

    return result === "" ? "EMPTY" : result;
}

File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.py
def solution(code):
    answer = ''
    mode = 0
    
    for idx in range(len(code)):
        char = code[idx]
        
        if mode == 0:
            if char == "1":
                mode = 1
            elif idx % 2 == 0:
                answer += char
        else:
            if char == "1":
                mode = 0
            elif idx % 2 == 1:
                answer += char
    
    return answer if answer != '' else 'EMPTY'